src/resources/dashboard.yml

```diff
@@ -13,7 +13,7 @@ dashboard:
     in-review:
       message_type: success
       title: Your application is being reviewed…
-      subline: Sit back and relax — we’ll get back to you as soon as we can.
+      subline: Sit back and relax—we’ll get back to you as soon as we can.
```

Comment:
I'm really not a fan of this style, and I tend to see spaces around the em dash much more frequently in blogs and in print.  Maybe it's just me though (@moosichu, what do you think?)

Reply:
imo maybe we should change it, but it shouldn't block this merge.